### DISCUSSION TITLE
how to keep old max stack while writing the module

### DISCUSSION THREAD

**Discussion by HPXCoder** [Washi1337/AsmResolver#251] [Q&A] [answered]
how to keep old max stack while writing the module?

**Comment by Washi1337** [comment, marked as answer]
This can be done for specific methods, using the `CilMethodBody.ComputeMaxStackOnBuild` property, or applied to the entire module by configuring the `CilMethodBodySerializer` in the builder options.

References and examples:
- https://asmresolver.readthedocs.io/en/latest/dotnet/managed-method-bodies.html#maximum-stack-depth
- https://asmresolver.readthedocs.io/en/latest/dotnet/advanced-pe-image-building.html#preserving-maximum-stack-depth

**Reply by HPXCoder** [reply]
THANKS
